// containers/moto/index.js
import React, {Component} from 'react'
import {connect} from 'react-redux'
import R from 'ramda'
import {Link} from 'react-router'

import {getMotoById} from 'selectors'
import {fetchMotoById, addMotoToBasket} from 'actions'
import BasketCart from 'components/basketCart'

class Moto extends Component {
  componentDidMount () {
    this.props.fetchMotoById(this.props.params.id)
  }

  renderFields () {
    const {moto} = this.props
    const columnFields = R.compose(
      R.toPairs,
      R.pick([
        'cpu',
        'camera',
        'size',
        'weight',
        'display',
        'battery',
        'memory'
      ])
    )(moto)

    return columnFields.map(([key, value]) => (
      <div className='column' key={key}>
        <div className='ab-details-title'>
          <p>{key}</p>
        </div>
        <div className='ab-details-info'>
          {value}
        </div>
      </div>
    ))
  }

  renderContent () {
    const {moto} = this.props

    return (
      <div className='thumbnail'>
        <div className='row'>
          <div className='col-md-6'>
            <img
              className='img-thumbnail'
              src={moto.image}
              alt={moto.name}
            />
          </div>
          <div className='col-md-6'>
            {this.renderFields()}
          </div>
        </div>
        <div className='caption-full'>
          <h4 className='pull-right'>${moto.price}</h4>
          <h4>{moto.name}</h4>
          <p>{moto.description}</p>
        </div>
      </div>
    )
  }

  renderSidebar () {
    const {moto, addMotoToBasket} = this.props
    return (
      <div>
        <p className='lead'>Quick shop</p>
        <BasketCart />
        <div className='form-group'>
          <h1>{moto.name}</h1>
          <h2>${moto.price}</h2>
        </div>
        <Link to='/' className='btn btn-info btn-block'>Back to store</Link>
        <button
          type='button'
          className='btn btn-success btn-block'
          onClick={() => addMotoToBasket(moto.id)}
        >
          Add to cart
        </button>
      </div>
    )
  }

  render () {
    const {moto} = this.props
    return (
      <div className='view-container'>
        <div className='container'>
          <div className='row'>
            <div className='col-md-9'>
              {moto && this.renderContent()}
            </div>
            <div className='col-md-3'>
              {moto && this.renderSidebar()}
            </div>
          </div>
        </div>
      </div>
    )
  }
}

const mapStateToProps = state => {
  return {
    moto: getMotoById(state, state.motoPage.id)
  }
}

const mapDispatchToProps = {
  fetchMotoById,
  addMotoToBasket
}

export default connect(mapStateToProps, mapDispatchToProps)(Moto)
