// api/mockMotos.js
export default [
  {
    id: "1",
    categoryId: "1",
    name: "MT-10",
    description: "Мощность. Крутящий момент. Маневренность.",
    price: 10000,
    image: "/uploads/mt-10.jpeg",
    "Тип двигателя": " 4-тактный, 4-цилиндровый, 4-клапанный",
    "Объем": "998 cm³",
    "Мощность": "118,0 кВт (160,4 л.с.) при 11 500 об. / мин",
    "Снаряженная масса": "210 кг",
    "Главная передача": "Цепь",
    "Система передней подвески": "Телескопическая",
    "Система задней подвески": "рычажная подвеска"
  },
  {
    id: "2",
    categoryId: "1",
    name: "MT-7",
    description: "Тяжело найти человека, который не любил бы новинки. Будь это новый телефон или предмет интерьера. Это же относится и к любителям «железных коней». Именно поэтому, мы хотим представить вам очередной шедевр в мире мотоциклов – модель Yamaha MT-07.",
    price: 8230,
    image: "/uploads/mt-7.jpeg",
    "Тип двигателя": "2-цилиндровый",
    "Объем": "689 cm³",
    "Мощность": "55.0 кВт (74.8 л.с.) @ 9,000 об./мин",
    "Снаряженная масса": "179 кг",
    "Главная передача": "Цепь",
    "Система передней подвески": "Телескопическая",
    "Система задней подвески": "Маятниковая"
  },
  {
    id: "3",
    categoryId: "2",
    name: "YZF-R1M",
    description: "С приходом немецкого спортбайка BMW S1000RR «Ямаха» R1 начала сдавать позиции, а пилоты все чаще делали выбор в пользу более современного и «электронного» баварского мотоцикла. Разумеется, господство БМВ не могло длиться вечно, и в 2015 году японцы показали свой ответ – новый спортбайк Yamaha YZF-R1 и ее заряженную версию R1M, отличающуюся большей мощностью и меньшей массой, которую удалось снизить за счет применения алюминия и карбона. Последняя стала конкурентом «заряженному» спортбайку S1000RR HP4, но фактически официальные дилеры Yamaha в России продают только ее, а перспективы появления бюджетной «просто» R1 пока туманны.",
    price: 13500,
    image: "/uploads/r1m.jpeg",
    "Тип двигателя": "4-цилиндровый",
    "Объем": "998 куб. см",
    "Мощность": "147,1 кВт (200,0 л.с.) при 13 500 об. / мин",
    "Снаряженная масса": "201 кг",
    "Главная передача": "Цепь",
    "Система передней подвески": "Телескопическая",
    "Система задней подвески": "маятниковая"
  },
  {
    id: "4",
    categoryId: "2",
    name: "YZF-R3",
    description: "Легендарная R-серия спортбайков YAMAHA устанавливает новые стандарты в своём классе благодаря продвинутым технологиям, впечатляющему дизайну и невероятной производительности.Наконец-то появилась модель, которая идеально заполняет промежуток между доступным R125 и чемпионским R6: абсолютно новый байк YZF-R3!",
    price: 9650,
    image: "/uploads/r3.jpeg",
    "Тип двигателя": "2-цилиндровый",
    "Объем": "321 куб. см",
    "Мощность": "30.9 кВт (42 л.с.) при 10 750 об. / мин",
    "Снаряженная масса": "169 кг",
    "Главная передача": "Цепь",
    "Система передней подвески": "Телескопическая",
    "Система задней подвески": "Маятниковая"
  },
  {
    id: "5",
    categoryId: "3",
    name: "XSR900",
    description: "XSR900 сочетает в себе лучшее из прошлого и настоящего. Захватывающий опыт вождения в сочетании с современными технологиями и стилем, который сформировался под влиянием непревзойденного участия в гонках. Новейшие технологии Yamaha упакованы в ретро-дизайн, чтобы отдать дань вечным классическим моделям.",
    price: 8230,
    image: "/uploads/xsr900.jpeg",
    "Тип двигателя": "3-цилиндровый",
    "Объем": "847 куб.см",
    "Мощность": "84.6 кВт (115 л.с.) при 10,000 об./мин",
    "Снаряженная масса": "188 кг / ABS 191 кг",
    "Главная передача": "Цепь",
    "Система передней подвески": "Телескопическая",
    "Система задней подвески": "Рычажная"
  },
  {
    id: "6",
    categoryId: "3",
    name: "XV950R",
    description: "Теперь Yamaha открывает дверь в мир кастомайзинга каждому владельцу нового XV950R – «боббера», который воплощает характер и личность оригинальных дорожных мотоциклов в современном японском нео-ретро стиле.",
    price: 11620,
    image: "/uploads/xv950.jpg",
    "Тип двигателя": " V-образный с 2 цилиндрами",
    "Объем": "942 куб.см",
    "Мощность": "38.3 кВт (52.1 л.с.)",
    "Снаряженная масса": "251 кг",
    "Главная передача": "Ремень",
    "Система передней подвески": "Телескопическая",
    "Система задней подвески": "Маятниковая"
  },
  {
    id: "7",
    categoryId: "4",
    name: "FJR1300",
    description: "За последние десять лет этот поистине универсальный мотоцикл дал тысячам владельцев возможность испытать ощущения как от класса Суперспорт и оценить по достоинству непревзойденный комфорт.",
    price: 15500,
    image: "/uploads/fjr1300.jpeg",
    "Тип двигателя": "4-цилиндровый",
    "Объем": "	1 298 куб. см",
    "Мощность": "107.5 кВт (146.2 л.с.) при 8 000 об. / мин",
    "Снаряженная масса": "289 кг",
    "Главная передача": "Вал",
    "Система передней подвески": "Телескопическая",
    "Система задней подвески": "Маятниковая"
  },
  {
    id: "8",
    categoryId: "4",
    name: "FZ1-S",
    description: "Модель нейкеда Yamaha FZ-1 появилась в 2001 году на базе классической стальной рамы дорожника и дефорсированного карбюраторного двигателя от Yamaha YZF-R1.",
    price: 10250,
    image: "/uploads/fz1-s.jpeg",
    "Тип двигателя": "4-цилиндровый",
    "Объем": "998 см3",
    "Мощность": "150 л.с.	(110 кВт) при 11 000 об/мин",
    "Снаряженная масса": "226 кг",
    "Главная передача": "Цепь",
    "Система передней подвески": "Телескопическая",
    "Система задней подвески": "Маятниковая"
  },
  {
    id: "9",
    categoryId: "5",
    name: "V-MAX",
    description: "Модель пауэр-круизера Yamaha V-max 1200 была представлена в 1985 году, появившись впервые на рынке Северной Америки. Концепция мотоцикла представляла собой черты круизера, мощный 4-цилиндровый V-образный двигатель, а также стиль и эргономику дрэг-байка. V-MAX 1200 стал одним из первых серийных дрэгстеров классического внешнего вида, показывающим выдающиеся динамические характеристики в гонках на 402 метра.",
    price: 7500,
    image: "/uploads/v-max.jpeg",
    "Тип двигателя": "4-цилиндровый",
    "Объем": "1198 см³",
    "Мощность": "145,0 л.с.",
    "Снаряженная масса": "283 кг",
    "Главная передача": "Кардан",
    "Система передней подвески": "Телескопическая",
    "Система задней подвески": "Маятниковая"
  },
  {
    id: "10",
    categoryId: "5",
    name: "Star Venture",
    description: "Компания Yamaha представила роскошный туристический мотоцикл Yamaha Star Venture, который является отличной альтернативой тяжелым круизерам H-D, Indian и Honda Gold Wing. Новый мотоцикл комплектуется 1854-кубовым V-образным мотором, который является улучшенной версией двигателя Yamaha Roadliner/Stratoliner. ",
    price: 20000,
    image: "/uploads/Star.jpg",
    "Тип двигателя": "4-цилиндровый",
    "Объем": "1854 см3",
    "Мощность": "150 л.с.",
    "Снаряженная масса": "434 кг",
    "Главная передача": "Ремень",
    "Система передней подвески": "Телескопическая",
    "Система задней подвески": "Маятниковая"
  }
]
